tools/api-markdown-documenter/src/api-item-transforms/configuration/Configuration.ts

```diff
@@ -13,19 +13,18 @@ import {
 	getDocumentationSuiteOptionsWithDefaults,
 } from "./DocumentationSuiteOptions.js";
 import {
-	type ApiItemTransformationOptions,
-	getApiItemTransformationOptionsWithDefaults,
-} from "./TransformationOptions.js";
+	type ApiItemTransformations,
+	getApiItemTransformationsWithDefaults,
+} from "./Transformations.js";
 
 /**
- * API Item transformation configuration.
+ * Shared base type for {@link ApiItemTransformationConfiguration} and {@link ApiItemTransformationOptions}.
+ *
+ * @remarks Not intended to be used directly.
```

Comment:
`@system` + `@sealed`?

Reply:
This library doesn't support the `@system` tag - that's a concept specific to the client release group. It probably makes sense more generally, but the tooling won't support it here yet.

But `@sealed` is a good call. Will update.